iroha_data_model/src/query.rs

```diff
@@ -73,8 +73,6 @@ pub enum QueryBox {
     FindDomainByName(FindDomainByName),
     /// `FindAllPeers` variant.
     FindAllPeers(FindAllPeers),
-    /// `FindAllParameters` variant.
-    FindAllParameters(FindAllParameters),
```

Comment:
Is it ok?

Reply:
Yes, I removed parameters impl for now as they are useless and have no implementation

Reply:
We decided to postpone this task with runtime config update